src/actions/index.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+
+const ROOT_URL='https://yts.am/api/v2/list_movies.json';
```

Comment:
ROOT_URL should be 'https://yts.am/api/v2'
Because you'll be able to reuse it later if you have some other endpoint for example `${ROOT_URL}/users`